README.md

```diff
@@ -287,6 +289,29 @@ The `assertionId` is appended to the `revocationList` URL request as query param
 $ npm run test
 ```
 
+### Manual testing
+You can test the build output (`npm run build`) in the browser and/or in nodejs to ensure you get satisfactory results with low overhead.
+
+#### Browser
+To test the browser build simply run `npm run test:manual:browser`. This will open an HTML page if your default browser which will test the fixtures for each version.
+
+NOTE V1: At the time of writing this, it is expected that the v1 certificate will fail on retrieving the issuer profile due to a mixed content block from your browser.
+To circumvent the issue you can either modify the v1 certificate to set the issuer profile to HTTPS. This means tempring with the certificate and will yield a hash mismatch error.
+You may also change the settings of your browser to disable the Mixed Content Block filter. The certificate still does not validate since it's been revoked.
+We are looking for a working v1 example.
```

Comment:
So about this: I've looked everywhere but couldn't find one. I wonder if we should re-issue a v1, for old times sake.